internal/gen3fuse.go

```diff
@@ -132,8 +132,13 @@ func InitializeInodes(didToFileInfo map[string]*indexdResponse) map[fuseops.Inod
 	
 	k := 0
 	for did, fileInfo := range didToFileInfo {
-		var filename = fileInfo.Filename  // TODO: Determine whether the true filename is preferable, or just the DID
-
+		if fileInfo.Filename == "" {
```

Comment:
where did you get this filename? if from indexd record file_name, this shouldn't be required to fuse mount it